repository_instrument.go

```diff
@@ -403,6 +429,74 @@ func (r *instrumentRepository) DeleteInstrument(ctx context.Context, id uuid.UUI
 	return nil
 }
 
+func (r *instrumentRepository) CreateFtpConfig(ctx context.Context, ftpConfig *FTPConfig) error {
```

Comment:
I think the FTPConfig parameters in this (and subsequent) method(s) do not need to be pointer types,

Reply:
Changed it to struct.